minecraft/protocol/packet/start_game.go

```diff
@@ -190,6 +190,8 @@ type StartGame struct {
 	// is a new system introduced in 1.16. Backwards compatibility with the inventory transactions has to
 	// some extent been preserved, but will eventually be removed.
 	ServerAuthoritativeInventory bool
+	// GameVersion is the version of the game the server is running. The exact function of this field isn't clear.
+	GameVersion string
```

Comment:
This exists side by side with the BaseGameVersion?

Reply:
Yeah, it's a bit... weird, but that's what it seems to be.